src/Managed/GoogleAnalytics.Core/HitBuilder.cs

```diff
@@ -78,7 +78,7 @@ public static HitBuilder CreateException(string description, bool isFatal)
 
             var data = new Dictionary<string, string>();
             data.Add("t", HitType_Exception);
-            if (description != null) data.Add("exd", description);
+            data.Add("exd", description);
```

Comment:
exd [Exception Description](https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters#exd) is not a required field for the exception hit.    In fact it is only valid for the exception hit there for it can be null.

Debug test it:

> https://www.google-analytics.com/debug/collect?tid=UA-6666666-1&v=1&cid=555&t=exception

Response:

```
{
  "hitParsingResult": [ {
    "valid": true,
    "parserMessage": [ ],
    "hit": "/debug/collect?tid=UA-6666666-1\u0026v=1\u0026cid=555\u0026t=exception"
  } ],
  "parserMessage": [ {
    "messageType": "INFO",
    "description": "Found 1 hit in the request."
  } ]
}
```

While I may agree that logically it would make sense to include a description to your exception.    If the measurement protocol does not require it then I don't think we should either.